ximea_ros2_cam/src/ximea_ros2_cam.cpp

```diff
@@ -165,6 +185,13 @@ void XimeaROSCam::openCam() {
                             XI_PRM_IMAGE_DATA_FORMAT,
                             this->cam_format_int_);
 
+    // Downsampling config
+    xi_stat = xiSetParamInt(this->xi_h_, XI_PRM_DOWNSAMPLING, this->cam_downsampling_int_);
+    xi_stat = xiSetParamInt(this->xi_h_, XI_PRM_DOWNSAMPLING_TYPE, this->cam_downsampling_type_int_);
+    RCLCPP_INFO_STREAM(this->get_logger(),  "Setting downsampling to: " << this->cam_downsampling_ << " type:" << this->cam_downsampling_type_);
+
+    RCLCPP_INFO_STREAM(this->get_logger(),  "Status: " << xi_stat);
```

Comment:
I create a macro called LOG_IF to be able to disable by parameter the debug mode (because the debugs was already in the package, and can be helpful considering the high number of camera configs). Commit: https://github.com/Brazilian-Institute-of-Robotics/ximea_ROS2_driver/commit/2bb6e0eae9117872aa357f1acf15b0cc064acba2